tests/api/contact.test.js

```diff
@@ -25,8 +25,8 @@ describe('contact handler', () => {
 
   test('should return 422 error when parameters missing', async () => {
     const body = {
-      name: '', // not valid
-      email: 'jody@mail.com',
+      name: 'Jody',
+      email: '', // not valid
```

Comment:
Please add test coverage for the smaller set of the form body being passed to this endpoint.

Reply:
I thought I did that by switching the empty input from name to email. If that isn't the case, would you be able to guide me on how you need it to be tested?